Resources/Public/JavaScript/GUI.js

```diff
@@ -268,13 +268,21 @@ define(['jquery', 'TYPO3/CMS/FrontendEditing/Crud', 'TYPO3/CMS/FrontendEditing/E
 
 		deferred.done(function () {
 			Editor.init($iframe, editorConfigurationUrl, resourcePath);
+
+			if (sessionStorage.scrollTop != "undefined") {
```

Comment:
Use an absolute comparison !== instead of a relative one. Also use single quotes for the strings.